desktop/desktop/lib/state/README.md

```diff
@@ -0,0 +1,6 @@
+State
```

Comment:
Thanks for taking a look at this so quickly @mtias, for changes like this - what do you think a good strategy should be? Making them here, as part of this PR, or in `wp-desktop` before migrating it over?
To me making the changes in `wp-desktop` first seems logical, but we would lose some git history in doing so.
I'm also concerned with making too many changes to the structure while `wp-desktop` is being maintained as any deviations might cause headaches as changes are made there that need to be echoed here.